src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2036,11 +2039,29 @@ private static ParameterSymbol GetParameter(int argIndex, MemberAnalysisResult r
 
             // NB: OriginalDefinition, not ConstructedFrom.  Substitutions into containing symbols
             // must also be ignored for this tie-breaker.
+            var m1Original = m1.LeastOverriddenMember.OriginalDefinition.GetParameters();
+            var m2Original = m2.LeastOverriddenMember.OriginalDefinition.GetParameters();
+
+            // Prefer params Span<T> or ReadOnlySpan<T> over params T[].
```

Comment:
The spec says there's an order of preference: `ReadOnlySpan<T>` first, then `Span<T>`, `T[]` and `IEnumerable<T>`